server/modules/turf/turf.controller.ts

```
@@ -146,99 +176,133 @@
+          400,
+        ),
+      );
+    }
   }
 
-  // Field-specific parser functions
-  private parseBasePrice(value: any, next: NextFunction) {
+  // Update parser functions to not require next parameter
+  private parseBasePrice(value: any) {
     if (!value) return { data: {} };
     const parsed = parseFloat(value);
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)